Utils.lua

```diff
@@ -615,6 +615,23 @@ function UTILS.LibStModifierFnNumber(a1, b1)
     return (tonumber(RemoveColorCode(a1)) or 0), (tonumber(RemoveColorCode(b1)) or 0)
 end
 
+function UTILS.LibStModifierFnVersion(a1, b1)
```

Comment:
Could you add in a comment above the function what it does?
```lua
--[[
    Transforms a version string of the form 3.4.5 to a number 345
    ]]--
````

Reply:
The function transforms a version string of the form 3.4.5 to the number 30405 (major*10000+minor*100+patch*1)